src/main/java/com/google/cloud/spanner/myadapter/session/SystemVariable.java

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.cloud.spanner.myadapter.session;
+
+import com.google.api.core.InternalApi;
+import com.google.cloud.spanner.SpannerException;
+import com.google.cloud.spanner.Type;
+import com.google.cloud.spanner.myadapter.utils.Converter;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import java.util.*;
+import javax.annotation.Nonnull;
+
+/** Represents a row in the pg_settings table. */
+@InternalApi
+public class SystemVariable {
+  private static final int NAME_INDEX = 0;
+  private static final int DATATYPE_INDEX = 1;
+  private static final int VALUE_INDEX = 2;
+
+  private String extension;
```

Comment:
What is the role of this?

Reply:
This got copied over from PGAdapter, but is not needed now. I mentioned in our last conversation that I'll remove it as part of the use defined variable implementation, but let me go ahead and clean up now itself to avoid confusion.